internal/utils/config.go

```diff
@@ -229,21 +276,12 @@ func LoadConfigFS(fsys afero.Fs) error {
 		if Config.Api.Port == 0 {
 			return errors.New("Missing required field in config: api.port")
 		}
-		if Config.Api.MaxRows == 0 {
-			Config.Api.MaxRows = 1000
-		}
```

Comment:
These need to be kept - setting the field in init_config.toml doesn't change the fact that these are optional and have default values.

Reply:
Also, we can't detect if boolean values are/aren't supplied if we use `bool` instead of `*bool`.

Reply:
I made a change on L234 to load init_config before user's config.toml. This allows us to define default values in one place.

Reply:
Ah, missed that